llvm/test/CodeGen/RISCV/rvv/fixed-vectors-masked-gather.ll

```diff
@@ -15086,22 +15086,18 @@ define <32 x i64> @mgather_strided_split(ptr %base) {
   ret <32 x i64> %x
 }
 
-; FIXME: This is a miscompile triggered by the mgather ->
-; riscv.masked.strided.load combine. In order for it to trigger we need either a
-; strided gather that RISCVGatherScatterLowering doesn't pick up, or a new
-; strided gather generated by the widening sew combine.
 define <4 x i32> @masked_gather_widen_sew_negative_stride(ptr %base) {
 ; RV32V-LABEL: masked_gather_widen_sew_negative_stride:
 ; RV32V:       # %bb.0:
-; RV32V-NEXT:    addi a0, a0, -128
+; RV32V-NEXT:    addi a0, a0, 128
 ; RV32V-NEXT:    li a1, -128
```

Comment:
I think I would like to see something closer to the original test case where we also calculated the wrong stride. Here the stride was correct.

Reply:
I've updated the test in 11d115d0569b212dfeb7fe6485be48070e068e19 to make the stride larger than 128